Sources/PreviewSnapshotsTesting/PreviewSnapshots+assertSnapshots.swift

```diff
@@ -11,9 +11,11 @@
 // or implied. See the License for the specific language governing permissions and limitations under
 // the License.
 
+#if canImport(UIKit)
```

Comment:
I really hope watchOS support gets approved. Should this if import just wrap the UIKit import and why is this needed?

Reply:
Because UIImage is used and UIKit import is missing.